BerriAI/litellm
# [Test] Add better testing for /key/generate, /user/new

## Testing Added for


 1. Generate a Key, and use it to make a call
 2. Make a call with invalid key, expect it to fail
 3. Make a call to a key with invalid model - expect to fail
 4. Make a call to a key with valid model - expect to pass
 5. Make a call with expired key - expect to fail
 6. Make a call with unexpired key - expect to pass
 7. Make a call with key under budget, expect to pass
 8. Make a call with key over budget, expect to fail
